### ISSUE TITLE
Pagination with group by not working for laravel ^7

### ISSUE THREAD

**Issue by deandex** [yajra/laravel-oci8#613]
### Summary of problem or feature request

In Laravel ^7 there is an aggregate expression when we running pagination with groups or having. And this is not supported by oracle query. And this feature has been added in laravel ^7. I think we need customize pagination function for laravel ^7 that not added groups or having checking condition.

### Code snippet of problem

```php
$without = $this->unions ? ['orders', 'limit', 'offset'] : ['columns', 'orders', 'limit', 'offset'];

        return $this->cloneWithout($without)
                    ->cloneWithoutBindings($this->unions ? ['order'] : ['select', 'order'])
                    ->setAggregate('count', $this->withoutSelectAliases($columns))
                    ->get()->all();
```

We only need that code for pagination.

### System details

- Operating System : Ubuntu 18.04.5 LTS
- PHP Version : 7.4
- Laravel Version : 7
- Laravel-OCI8 Version : 7

**Comment by deandex** [comment]
I have fork the project and create my custom pagination function without alias table. If okay, i can make a pull request for this issue

**Comment by yajra** [comment]
Fix on v7.0.2 Thanks!
